### ISSUE TITLE
Undo option added?

### ISSUE THREAD

**Issue by switchmaxfx** [maruohon/litematica#658]
Is there a option to undo a placement using this mod?

**Comment by maruohon** [comment]
No, not in the current versions. Undo functionality is still only on the todo list for the mod...

**Comment by mikaba-eu** [comment]
would be nice to get this. we use litematica a lot in creative, instant of worldedit atm

**Comment by DOTLeo0913** [comment]
Plz add undo
